index.js

```diff
@@ -68,8 +69,8 @@ Deployer.prototype.isReady = function(version) {
   return Promise.all(deferreds);
 };
 
-// imagePath is optional, and can be carried through the function calls as 'undefined'
-Deployer.prototype.deploy = function(version, imagePath) {
```

Comment:
I dont think so... It was undocumented and was only available for a brief period.